runtime/src/bank.rs

```diff
@@ -4538,7 +4544,7 @@ impl Bank {
         );
         let native_loader = native_loader::id();
         for builtin_program in self.builtin_programs.iter() {
-            program_accounts_map.insert(*builtin_program, &native_loader);
+            program_accounts_map.insert(*builtin_program, (&native_loader, 1));
```

Comment:
Built-in programs which are not explicitly listed in TX should not increase the counter, so it should be 0 here.